packages/docs/site/docs/01-start-here/02-overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Overview
+slug: /overview
+---
+
+# Overview
+WordPress Playground is an online platform that allows you to experiment and learn about WordPress without affecting your live website. It's like a virtual sandbox where you can play around with different features, designs, and settings in a safe and controlled environment.
+
+## Here's how it works
+When you first start using WordPress Playground, you'll be provided with a separate space where you can create and customise your own WordPress website. This space is completely isolated from your actual website, so any changes you make here won't affect your live site.
+
+## Try themes and plugins on the fly
+Within the WordPress Playground, you can explore various themes, which are pre-designed templates that determine the appearance of your website. You can choose from a wide range of themes and see how they look on your site. You can also modify the colors, fonts, layouts, and other visual elements to create a unique design.
+In addition to themes, you can experiment with plugins too. With WordPress Playground, you can install and test different plugins to see how they work and what they can do for your site. This allows you to explore and understand the capabilities of WordPress without worrying about breaking anything.
+
+## Try content on the run
+Another great feature of WordPress Playground is the ability to create and edit content. You can write blog posts, create pages and add media like images & videos to your site. This helps you understand how to organise and structure your content effectively.
+
+## And, yes it's safe
+Overall, WordPress Playground provides a risk-free environment for beginners to learn and get hands-on experience with WordPress. It helps you to gain confidence and knowledge before making changes to your live website.
+
+Remember, anything you do in the WordPress Playground will not impact your actual website, so feel free to explore, try new things, and have fun learning about WordPress!
```

Comment:
> will not impact your actual website

This point is really stressed throughout the document, maybe it's enough to say it once?

Reply:
I have removed it.